eve/main.yml

```diff
@@ -37,4 +37,4 @@ stages:
           workdir: '%(prop:builddir)s/build'
       - ShellCommand:
           name: run feature tests
-          command: npm run ft_test
+          command: TEST_SWITCH=1 npm run ft_test
```

Comment:
You can introduce env `CI=true` globally. This will be consistent with CircleCI for opensource. See https://github.com/scality/S3/blob/development/8.0/eve/main.yml for reference.

Reply:
I was actually relying on an existing variable, but I agree a global CI variable would make sense.